consensus/leader.go

```diff
@@ -262,20 +262,19 @@ func (consensus *Consensus) onCommit(msg *msg_pb.Message) {
 		return
 	}
 
+	viewID := consensus.viewID
+
 	quorumIsMet := consensus.Decider.IsQuorumAchieved(quorum.Commit)
 	if !quorumWasMet && quorumIsMet {
 		logger.Info().Msg("[OnCommit] 2/3 Enough commits received")
-		go func(viewID uint64) {
```

Comment:
This logic shouldn't be changed.

Reply:
The new code is already merged in master branch, so we shall also revert it in master branch.

Reply:
I see no difference between the original code and the new one. Checked timestamp of HOORAY, 8s as well.

Reply:
ok, it wasn't commited to t3 to avoid risk on the launch. If now it tested fine. We could merge it. Did you check the Grace period to be 2 seconds? 8 second block time is not related to this part of the code. Only the grace period of 2 seconds.

Reply:
okay, I see, I checked the grace period and found it is more than 2s actually. And I was actually confused.

Reply:
https://github.com/harmony-one/harmony/issues/3096, deep dive, and found no issue.